docs/gke/configs/env-kubeflow.sh

```diff
@@ -4,11 +4,15 @@
 # This is script defines values for all the variables used in
 # the instructions.
 
-# Bucket and project must be unique for each project 
+# Bucket and project must be unique for each project
 
 # Set PROJECT to the project you want to use with Kubeflow.
 export PROJECT=<your_project>
 
+# This should be the cluster admin's email address. This is used for
+# OAuth Login screen.
+export OAUTH_EMAIL_ADDRESS=user@example.com
```

Comment:
Just make this a parameter on the ksonnet component and set the environment variable directly on the K8s deployment without creating a configmap